src/main/java/no/nav/familie/integrasjoner/config/ApiExceptionHandler.kt

```diff
@@ -91,6 +92,21 @@ class ApiExceptionHandler {
             .body(failure("""Det oppstod en feil. ${e.message}""", error = e))
     }
 
+    // ValueRetrievalException har key (ofte ident) i feilmelding -> må logges til securelog
+    @ExceptionHandler(ValueRetrievalException::class)
+    fun handleException(e: ValueRetrievalException): ResponseEntity<Ressurs<Any>> {
```

Comment:
Hadde det kanskje vært fint å kun endre handlern for `Exception`? For å unngå at fnr logges for denne og andre exceptions? Då trenger vi ikke en spesifik handler for denne exception
 stacktrace logges alltid der
`logger.error("Exception : {} {}", e.javaClass.name, e.message, e)`